tests/paymentTimeout.test.ts

```
@@ -0,0 +1,168 @@
+    ]);
+
+    console.log("Transaction mined:", receipt);
+    return receipt;
+  } catch (error) {
+    console.error("Transaction failed:", error);
+    throw error;
+  } finally {
+    // Clean up any remaining timeout promises
+    await vi.runAllTimersAsync();
+  }
+}
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)